src/schema/schema_create.c

```diff
@@ -859,11 +865,13 @@ __create_table(WT_SESSION_IMPL *session, const char *uri, bool exclusive, const
             __wt_config_init(session, &conf, config);
             for (nkeys = 0; (ret = __wt_config_next(&conf, &ckey, &cval)) == 0; nkeys++)
                 ;
-            if (nkeys == 1)
+            if (nkeys == 1) {
+                WT_STAT_CONN_INCR(session, session_table_create_import_fail);
```

Comment:
Per my comment above I think we only want to indicate if the call succeeds or fails and remove/not have all these places where we increment the statistic.